include/seqan3/alphabet/quality/concept.hpp

```diff
@@ -109,8 +109,8 @@ concept bool quality_concept = requires(q quality)
     //! requires fulfillment of alphabet concept
     requires alphabet_concept<q>;
 
-    //! requires additionally public getter and setter for integral phred score representation
-    { from_phred(quality, typename q::integral_type{}) } -> q;
+    //! requires additionally public getter and setter for rank phred score representation
+    { assign_phred(quality, typename q::rank_type{}) } -> q;
```

Comment:
concepts that are used in interfaces and that might be satisfied by external types should only require free functions, not member functions. At least everywhere where we get to define the concept ourselves.